pipelines/config/dependency-builds.yml

```diff
@@ -381,8 +382,9 @@ dependencies:
     versions_to_keep: 2
     copy-stacks:
       - bionic
-build_stacks: [ 'cflinuxfs3' ]
-copy_stacks: [ 'bionic' ]
+cflinuxfs4_dependencies: [ 'node' ]
+build_stacks: [ 'cflinuxfs4' , 'cflinuxfs3' ]
+copy_stacks: [ 'jammy', 'bionic' ]
 windows_stacks: [ 'windows2016', 'windows' ]
 deprecated_stacks: [ 'cflinuxfs2' ]
 v3_stacks:
```

Comment:
Also in the config, wouldn't we need some mechanism to filter out version-lines like dotnet-{sdk,runtime,asp} 3.1.X that would never build successfully on cflinuxfs4? (See Ryan's comment in the doc)

Reply:
Yeah, I just updated the PR with the proposed changes to skip building specific version lines in cflinuxfs4.